### PR TITLE
Testing linting on PR for backend 

### PR DESCRIPTION
**Pull request by HuangStanley050**

### CODE REVIEW

**Review by danieltongaus — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -4,7 +4,8 @@
   "description": "A Node.js starter for the Serverless Framework with async/await and unit test support",
   "main": "handler.js",
   "scripts": {
-    "test": "serverless-bundle test"
+    "test": "serverless-bundle test",
+    "lint": "serverless-bundle eslint"
```
Comment (danieltongaus) [inline package.json]: Might be better to move eslint before the test,
Just so you get earlier feedback, as the tests takes a few mins to run

Comment (jpeyper) [reply package.json]: npm doesnt run the scripts in the order they are defined.

**Review by jpeyper — CHANGES_REQUESTED** [review]
You don't want to be committing .env.gpg (I don't think...)

**Comment by HuangStanley050** [comment]
> You don't want to be committing .env.gpg (I don't think...)

That's part of the GitHub action from before, GitHub action will first decrypt and then serverless can pack the unencrypted .env before deploy

**Comment by jpeyper** [comment]
> > You don't want to be committing .env.gpg (I don't think...)
> 
> That's part of the GitHub action from before, GitHub action will first decrypt and then serverless can pack the unencrypted .env before deploy

I'm not seeing where that is happening in this PR, am I just missing it? shouldn't we be using GitHub secrets for this kind of thing?

**Comment by HuangStanley050** [comment]
> > > You don't want to be committing .env.gpg (I don't think...)
> > 
> > 
> > That's part of the GitHub action from before, GitHub action will first decrypt and then serverless can pack the unencrypted .env before deploy
> 
> I'm not seeing where that is happening in this PR, am I just missing it? shouldn't we be using GitHub secrets for this kind of thing?



> > > You don't want to be committing .env.gpg (I don't think...)
> > 
> > 
> > That's part of the GitHub action from before, GitHub action will first decrypt and then serverless can pack the unencrypted .env before deploy
> 
> I'm not seeing where that is happening in this PR, am I just missing it? shouldn't we be using GitHub secrets for this kind of thing?

.env.gpg has always been part of the master branch as serverless needs the .env from the file system to pack in the environment variables, I included .env.gpg in this PR because our AWS got nuked and I have to update the local .env file which was not checked into Github. Hence I had to re-encrypt the .env file locally and include an updated version of .env.gpg as this PR.

**Review by jpeyper — APPROVED** [review]

**Pull request merged**